### PR TITLE
Download stack analysis threshold value

### PR DESCRIPTION
**Pull request by tisnik**
Special step that is needed to get the stack analysis outlier probability threshold. That value is to be used in ongoing stack analysis v2 tests, and we don't want to use fixed magic constant in such tests.

### CODE REVIEW

File: integration-tests/features/steps/common.py [hunk 1]
```diff
@@ -744,3 +753,40 @@ def generate_authorization_token(context, private_key):
...
+    return response.text
+
+
+def parse_float_value_from_text_stream(text, key):
+    """Go through all lines of the text file, find the line with given key
+    and parse float value specified here"""
+    regexp = key + "\s*=\s*(\d.\d*)"
+    for line in text.split("\n"):
+        if line.startswith(key):
+            match = re.match(regexp, line)
```
Comment (fridex) [inline integration-tests/features/steps/common.py]: Probably `re.fullmatch`?

Comment (tisnik) [inline integration-tests/features/steps/common.py]: ah yes, that would be better, TY

**Comment by tisnik** [comment]
[test this]

**Comment by tisnik** [comment]
[test this]

**Comment by tisnik** [comment]
[test this]

**Comment by tisnik** [comment]
[test this]

**Comment by tisnik** [comment]
Big thanks to @jpopelka for fixing CI issue. I'm merging this change

**Pull request merged**